core/src/test/java/org/elasticsearch/index/query/BaseQueryTestCase.java

```diff
@@ -382,6 +387,65 @@ protected static String getRandomRewriteMethod() {
     }
 
     protected String getRandomType() {
-        return (currentTypes.length == 0) ? MetaData.ALL : randomFrom(currentTypes);  
+        return (currentTypes.length == 0) ? MetaData.ALL : randomFrom(currentTypes);
+    }
+
+    /**
+     * Helper method to return a random field (mapped or unmapped) and a value
+     */
+    protected static Tuple<String, Object> getRandomFieldNameAndValue() {
```

Comment:
wouldn't it be better to make the change only local to FuzzyQuery, and then have another PR in which we make use of this method for all the other queries? In which case I'm happy to move it back to FuzzyQueryBuilderTest.